faq/faq.rst

```diff
@@ -81,7 +81,7 @@ Note that it is not straightforward to change the shard count of an already dist
 How does citus support count(distinct) queries?
 -----------------------------------------------
 
-Citus can push down count(distinct) entirely down to the worker nodes in certain situations (for example if the distinct is on the distribution column or is grouped by the distribution column in hash-partitioned tables). In other situations, Citus uses the HyperLogLog extension to compute approximate distincts. You can read more details on how to enable approximate :ref:`count_distinct`.
+Yes, Citus can evaluate count(distinct) aggregates in -- and across -- worker nodes. It also supports the HyperLogLog extension to compute approximate distincts. You can read more details on how to enable approximate :ref:`count_distinct`.
```

Comment:
Can we link to the count-distinct section we modified above? Also, maybe we can expand on this to say something like:
a) It first tries to push down if its on the distribution column
b) Can pull distinct rows to master and calculate value or
c) Use approximate counts when (b) is too expensive.